api_tests/auth/test_endpoint.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import requests
+import json
+import config
+
+class TestEndpoint(unittest.TestCase):
+
+    @classmethod
+    def setUpClass (self):
+        config_dict  = config.parse()
+        self.API_URL = config_dict['url'] + '/auth'
+
+    def run_tests (self):
+        print '\nTesting /auth Endpoint:'
```

Comment:
I tried it using docstrings, but because of the way they are being run, the print statements are necessary to get the correct formatting and output.